aviary/interface/methods_for_level2.py

```diff
@@ -2451,7 +2522,13 @@ def _add_two_dof_landing_systems(self):
                 ("range_resid", Mission.Constraints.RANGE_RESIDUAL)],
         )
 
-    def _add_fuel_reserve_component(self, post_mission=True):
+    def _add_fuel_reserve_component(self, post_mission=True,
+                                    reserves_name=Mission.Design.RESERVE_FUEL):
+        if post_mission:
```

Comment:
We ultimately should start putting 2DOF downstream components into "post_mission", but that is for another story.

Reply:
It currently is part of the mission, because the amount of reserve fuel you carry will impact how much fuel you need in your primary mission, which will affect the reserve fuel required